### ISSUE TITLE
Fitting model with negataive samples only

### ISSUE THREAD

**Issue by alishan2040** [iqiukp/SVDD#14]
Hi,

Example no. 1 illustrates on how to train the SVDD model with negative samples only, but I see that both negative and positive samples are fed to the model during training stage. Is there a way to train the model only with negative samples? similar to how OC-SVM <https://scikit-learn.org/stable/modules/generated/sklearn.svm.OneClassSVM.html> is trained.

Thanks

**Comment by iqiukp** [comment]
Hi,

In general, the training set X_train of an SVDD model is provided without labels, and its training process uses svdd.fit(X_train) (please refer to the example svdd_example_unlabeled_data.py); if the training set X_train has the label y_train, its training process uses svdd. fit(X_train, y_train) (see example svdd_example_hybrid_data.py). The inclusion of negative class labels inside the training set enhances the description of positive class samples, but of course, negative class labels are not necessary for SVDD itself.
